jenkins/scripts/files/run_integration_tests.sh

```diff
@@ -121,9 +121,11 @@ else
   git checkout "${METAL3BRANCH}"
 fi
 
-if [[ "${TESTS_FOR}" == "feature_tests"* ]]
+if [[ "${TESTS_FOR}" == "upgrade_feature_tests"* ]]
 then
-  make ${TESTS_FOR}
+  make upgrade_test
```

Comment:
Root makefile target needs to be called here https://github.com/metal3-io/metal3-dev-env/blob/master/Makefile#L53
There will be another upgrade makefile target from https://github.com/metal3-io/metal3-dev-env/pull/453 so this is not scalable.

Reply:
Sorry I didn't know that you were adding a new target for centos based upgrade tests.